.github/CODEOWNERS

```diff
@@ -2,7 +2,24 @@
 # Rules are matched bottom-to-top, so one team can own subdirectories
 # and another team can own the rest of the directory.
 
-# All your base
+# API
+/datadog/api/           @DataDog/integrations-tools-and-libraries
```

Comment:
Small detail but could we align everything here?

Reply:
done